src/user/components/LoginForm/LoginForm.tsx

```diff
@@ -27,11 +27,18 @@ export const LoginForm = ({ handleFieldValueChange, handleSubmit, isSubmitDisabl
     event.preventDefault();
   };
 
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useLayoutEffect(() => {
+    inputRef.current?.focus();
+  }, []);
```

Comment:
Lets add one more useLayoutEffect that listens for errorMessage, and sets focus on username field if error message appears.